projects/plugins/protect/src/js/components/scan-footer/index.jsx

```diff
@@ -33,7 +33,7 @@ const ProductPromotion = () => {
 				<Title>{ __( 'Get access to our Cloud', 'jetpack-protect' ) }</Title>
 				<Text mb={ 3 }>
 					{ __(
-						'With your Protect upgrade, you have free access to scan your site on our Cloud, so you can be aware and fix your threats even if your site goes down. ',
+						'With your Protect upgrade, you have free access to scan your site on our Cloud, so you can be aware and fix your threats even if your site goes down.',
```

Comment:
I'm afraid we'll have to preserve this too for inline elements.

Reply:
What inline elements? The following element is a block-formatted button.

![Screenshot](https://github.com/user-attachments/assets/df01221a-5534-47bf-93d3-5cf8c219b6cd)